app/layout/simple.phtml

```diff
@@ -3,16 +3,17 @@
 	/** @var FreshRSS_View $this */
 	FreshRSS::preLayout();
 	$class = '';
+	$dir = '';
 	if (_t('gen.dir') === 'rtl') {
-		echo ' dir="rtl"';
+		$dir = ' dir="rtl"';
 		$class = 'rtl ';
 	}
 	if (FreshRSS_Context::userConf()->darkMode !== 'no') {
 		$class .= 'darkMode_' . FreshRSS_Context::userConf()->darkMode;
 	}
 ?>
 <!DOCTYPE html>
-<html lang="<?= FreshRSS_Context::userConf()->language ?>" xml:lang="<?= FreshRSS_Context::userConf()->language ?>" class="<?= $class ?>">
+<html lang="<?= FreshRSS_Context::userConf()->language ?>"<?= $dir ?> xml:lang="<?= FreshRSS_Context::userConf()->language ?>" class="<?= $class ?>">
```

Comment:
Note for later: it could be good to remove this file and use only `layout.phtml` if possible